my_compassion/controllers/my2_children.py

```diff
@@ -0,0 +1,76 @@
+##############################################################################
+#
+#    Copyright (C) 2025 Compassion CH (http://www.compassion.ch)
+#    @author: Noé Berdoz <nberdoz@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo.http import request
+from odoo import http
+
+
+class MyCompassionChildrenController(http.Controller):
+
+    @http.route('/my2/children/', type="http", auth="user", website=True)
```

Comment:
We don't want the MyCompassion pages to be indexed by search engines (I think).
```suggestion
    @http.route('/my2/children/', type="http", auth="user", website=True, sitemap=False)
```

Reply:
Thank you, related task created here [T2347](https://erp.compassion.ch/web#id=2939&action=521&active_id=901&model=project.task&view_type=form&cids=1&menu_id=2029)